src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1650,9 +1650,21 @@ public void testAllStyleRules() throws Exception {
                             fileName, ruleName);
                 }
 
-                validateStyleModules(XmlUtil.findChildElementsByTag(columns.get(2), "a"),
-                        XmlUtil.findChildElementsByTag(columns.get(3), "a"), styleChecks, styleName,
-                        ruleName);
+                final Iterator<Node> itrChecks =
+                        XmlUtil.findChildElementsByTag(columns.get(2), "a").iterator();
+                final Iterator<Node> itrConfigs =
+                        XmlUtil.findChildElementsByTag(columns.get(3), "a").iterator();
+
+                if ("google".equals(styleName)) {
+                    final Iterator<Node> itrSamples =
+                            XmlUtil.findChildElementsByTag(columns.get(4), "a").iterator();
+                    validateStyleModulesGoogle(itrChecks, itrConfigs, itrSamples,
+                            styleChecks, styleName, ruleName);
+                }
+                else {
+                    validateStyleModulesSun(itrChecks, itrConfigs,
+                            styleChecks, styleName, ruleName);
+                }
```

Comment:
Before this update, we had only one `validateStyleModules` method but in this PR I added a new column in google coverage page, so validateStyleModules method required to pass an extra parameter to it, sun style does not have this new column. So I had to create 2 different versions of validateStyleModules for both styling. I'm not sure if this is the correct way to distinguish both stylings, lemme know if there is another more efficient way to do this